examples/computer_vision/mnist_pl/mnist.py

```diff
@@ -15,14 +15,13 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import data
```

Comment:
I think this sorting of import is worse, since data should be treated as a local module rather than 3rd party (that is, it should below, as before).